addons/ConsoleBG/ConsoleBG.lua

```diff
@@ -28,19 +29,107 @@ config.register(settings, function(settings)
     windower.prim.set_size('ConsoleBG', settings.extents.x, settings.extents.y)
 end)
 
+windower.register_event('addon command', function(command1, command2, command3)
+
+    command1 = command1 and command1:lower() or 'help'
+    command2 = command2 and command2:lower()
```

Comment:
This will be `false` if the user doesn't provide it. Won't matter in this context, but not sure if this is intended. Would also be fixed by making an `args` table instead of explicitly declaring all arguments, especially since `command2` and `command3` aren't accurate terms, as they are not always commands.